src/screens/ContactScreen/index.js

```diff
@@ -365,6 +371,13 @@ const ContactScreen = ({navigation}) => {
       eventTrack.onCreateCommunityScreenUserNameUnselected();
       copyUsers.splice(indexSelectedUser, 1);
     } else {
+      if (copyUsers.length >= CONTACT_SCREEN_MAX_SELECTED_USERS) {
+        SimpleToast.show(
+          `You can only select up to ${CONTACT_SCREEN_MAX_SELECTED_USERS} users`,
+          SimpleToast.SHORT
+        );
+        return;
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A check has been added to limit the number of users that can be selected. This is a good practice as it prevents potential issues with excessive data or UI clutter. However, ensure that `CONTACT_SCREEN_MAX_SELECTED_USERS` is defined and has a reasonable value.
```diff
+    if (copyUsers.length >= CONTACT_SCREEN_MAX_SELECTED_USERS) {
+      SimpleToast.show(
+        `You can only select up to ${CONTACT_SCREEN_MAX_SELECTED_USERS} users`,
+        SimpleToast.SHORT
+      );
+      return;
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->